src/playback/view/toolbars/components/timecodemodel.cpp

```diff
@@ -238,20 +238,9 @@ void TimecodeModel::updateValueString()
     constexpr auto toNearest = true;
     QString newValueString = m_formatter->valueToString(m_value, toNearest).valueString;
 
-    if (newValueString.size() != m_valueString.size()) {
```

Comment:
the component works as expected on the main toolbar? please check it

Reply:
It does now. On your latest master, select the `hh:mm:ss + milliseconds` format, and then change to `bar:beat:tick`.

Reply:
for qa